src/classBased/InputTodo.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from "react"
+
+class InputTodo extends Component {
+  state = {
+    title: ""
+  }
+
+  handleOnChange = e => {
+    this.setState({
+      [e.target.name]: e.target.value
+    }) 
+  }
+
+  handleSubmit = e => {
+    e.preventDefault();
+    if (this.state.title.trim()) {
+      this.props.handleSubmitProps(this.state.title);
+    this.setState({
+      title: ""
+    })
+    }
+    else {
+      alert('Please write an item')
+    }
+    
```

Comment:
Linters are tools 💯 that perform static analysis on the software in order to recognize and report adherence or non-adherence to some set of coding best practices. So I would recommend you to use linters throughout your project 👌. Try to fix those linters errors and submit them for review again. 👍